### ISSUE TITLE
Design Addresses Database Schema

### ISSUE THREAD

**Issue by sreevishnu211** [CSCI-GA-2820-SP23-003/customers#23]
**As a** Customer Service Manager
**I need** to design an Addresses Schema
**So that** developers can create the address database object in PostgreSQL.
   
 ### Details and Assumptions
The Address model need to accommodate:
- Address ID (UNIQUE INC INT)
- Customer ID 
- Street
- City
- State
- Pincode
   
 ### Acceptance Criteria  
 ```gherkin
Given customer provides address information
When customer record is being created
Then the system should also create an address for the customer

Given customer doesn't provide address information
When customer record is being created
Then the system should request from the customer to provide information for at least one address
```

**Comment by sreevishnu211** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/customers-spring23-003-63e5271972c0b5edc5ad900d/issues/CSCI-GA-2820-SP23-003/customers/23?planning-poker) with Zenhub @Akshama @ayushjainid @mwj10 @sreevishnu211 @wannabeOG

**Comment by Akshama** [comment]
Let's move stories to Closed from Done once Sprint gets over.
Until then, keep it in Done so that our points are counted. :)

**Comment by Akshama** [comment]
I felt this issue involved a lot of collective efforts and time, should be an 8 instead of 5.

**Comment by ayushjainid** [comment]
I agree we can use this experience as a feedback for future story planning.

**Comment by ayushjainid** [comment]
Updated the Schema to include "Country" as a field too.
